storage/rocksdb/ha_rocksdb.cc

```diff
@@ -2785,9 +2792,8 @@ class Rdb_trx_info_aggregator : public Rdb_tx_list_walker {
       }
 
       std::string query_str;
-      LEX_STRING *const lex_str = thd_query_string(thd);
-      if (lex_str != nullptr && lex_str->str != nullptr) {
-        query_str = std::string(lex_str->str);
+      if (thd->query().str != nullptr) {
```

Comment:
lex_str I think can never be NULL as thd->query().str is an instance, not a pointer, but is it possible that the contents of .str can be {NULL, 0}?